packages/blockchain-link/src/workers/throttler.ts

```diff
@@ -0,0 +1,25 @@
+export class Throttler {
+    private delay: number;
+    private timeouts: { [id: string]: ReturnType<typeof setTimeout> };
+
+    constructor(delay: number) {
+        this.delay = delay;
+        this.timeouts = {};
+    }
+
+    throttle(id: string, callback: () => void) {
+        const previousTimeout = this.timeouts[id];
+        if (previousTimeout) {
+            clearTimeout(previousTimeout);
+        }
+        this.timeouts[id] = setTimeout(() => {
+            callback();
+            clearTimeout(this.timeouts[id]);
+            delete this.timeouts[id];
+        }, this.delay);
+    }
+
+    dispose() {
+        Object.values(this.timeouts).map(clearTimeout);
```

Comment:
delete all values in `this.timeouts` maybe?

Reply:
https://github.com/trezor/trezor-suite/pull/5454/commits/9987e3fc0304c40b2493acd5319f0f81643325f2 and I've added unit test to make sure `Throttler` works as expected.